### PR TITLE
[TASK] BEst practices

### PR DESCRIPTION
**Pull request by georgringer**

### CODE REVIEW

File: app/web/typo3conf/ext/theme/Configuration/TypoScript/Base/Config.setupts [hunk 1]
```diff
@@ -46,7 +46,7 @@ config {
 	headerComment = Created by Sup7even Digital :: http://www.sup7even.digital
 
 	debug = 0
-	admPanel = 0
```
Comment (jousch) [inline app/web/typo3conf/ext/theme/Configuration/TypoScript/Base/Config.setupts]: If there is no performance impact it makes sense to enable `config.admPanel`

File: app/web/typo3conf/AdditionalConfiguration_Development.php [hunk 2]
```diff
@@ -43,30 +43,10 @@
...
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['extbase_datamapfactory_datamap']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['extbase_object']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['extbase_reflection']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['extbase_typo3dbbackend_queries']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['l10n']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
-
-// News caches
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['cache_news_classes']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
-//$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['cache_news_category']['backend'] = 'TYPO3\CMS\Core\Cache\Backend\NullBackend';
+//$cacheConfigurations = $GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations'];
```
Comment (jousch) [inline app/web/typo3conf/AdditionalConfiguration_Development.php]: nice one!

File: app/.editorconfig [hunk 3]
```diff
@@ -20,25 +20,15 @@ indent_size = 4
 # HTML-Files
 [*.html]
 indent_style = tab
-indent_size = 4
-
-# TypoScript
-[{*.ts,*.setupts,*.constantsts,*.tsc,*.ts1}]
-indent_style = tab
-indent_size = 4
+indent_size = 2
```
Comment (jousch) [inline app/.editorconfig]: Did you just overwrite .editorconfig with the one from core? When yes, mine one is based on the core's with additionals. and I personally don't like only 2 indents for typoscript and html. Is it ok for you?

File: app/web/typo3conf/ext/theme/Configuration/TypoScript/Base/Config.setupts (hunk 1 above)
Comment (georgringer) [inline app/web/typo3conf/ext/theme/Configuration/TypoScript/Base/Config.setupts]: no and this is the only thing we have to "debug" a production environment

File: app/.editorconfig (hunk 3 above)
Comment (georgringer) [inline app/.editorconfig]: ok for me ;)

File: app/web/typo3conf/AdditionalConfiguration_Development.php (hunk 2 above)
Comment (jousch) [inline app/web/typo3conf/AdditionalConfiguration_Development.php]: Unfortunately this doesn't work as intended. @georgringer Any suggestion?

Comment (jousch) [inline app/web/typo3conf/AdditionalConfiguration_Development.php]: solved

**Comment by jousch** [comment]
Implemented with 9f1f565326f008cb09f3842327181f7903594500 92b52f84e86c3d506173b1fae185d9f33ee6ea2d f8e716b6c4ce20517730433b577df3146e6ae72e de3d979c2f17b164eb5bf0ab65a4f393ba38da58 9c96b5d7a071aa2a702b1dc82f9ccbced67bf052

**Pull request closed without merging**